.kokoro/build.bat

```diff
@@ -15,8 +15,4 @@
 :: downstream client libraries before they are released.
 :: See documentation in type-shell-output.bat
 
-set JAVA8_HOME=%JAVA_HOME:"=%
-choco install -y openjdk11
-set JAVA11_HOME=C:\Program Files\Eclipse Adoptium\jdk-11.0.18.10-hotspot\
-
```

Comment:
@rajatbhatta 
We generally update these jdk versions to latest manually, whenever a windows build fail. But do we need these explicitly getting installed and set JAVA11_HOME?

The build.sh file already sets these `JAVA8_HOME` and `JAVA11_HOME`
https://github.com/googleapis/java-spanner/blob/08af6714055b5770164db66ec3c54c3a35b58724/.kokoro/build.sh#L53-L54
https://github.com/googleapis/java-spanner/blob/08af6714055b5770164db66ec3c54c3a35b58724/.kokoro/build.sh#L28-L29

I compared the windows build by removing and having this and it seems it is able to capture the latest JDK by default.

`With out above lines`
https://screenshot.googleplex.com/8k9CiGKsgEknz8a

`With above lines after updating to latest version`
https://screenshot.googleplex.com/xBRpe9UXnmXjAPw

Can you please give your view here?

Reply:
CC: @mpeddada1 for their inputs on this..

Reply:
I have no idea why that is explicitly being installed here. But the outcome of the build seems to indicate that we don't need it.
Also; Our JDBC driver build setup also does not include it: https://github.com/googleapis/java-spanner-jdbc/blob/main/.kokoro/build.bat

Reply:
I can provide a bit of context here. The graal-sdk dependency, which is used to provide native image configurations requires a minimum version of JDK 11 otherwise it results in a compilation error:
```
  bad class file: C:\Users\kbuilder\.m2\repository\org\graalvm\sdk\graal-sdk\22.3.0\graal-sdk-22.3.0.jar(org/graalvm/nativeimage/hosted/Feature.class)
[ERROR]     class file has wrong version 55.0, should be 52.0
```
Since we still need to verify against JDK 8 for some jobs, this [change](https://github.com/googleapis/java-spanner/pull/1870/files#diff-6ec64aad727811781a78bc1dd7e4fd68ac708ccff197213425ae5933df4f0a49) was made to compile tests against JDK 11 but run them against JDK 8.  This logic requires both JAVA11_HOME and JAVA8_HOME env variables to be [provided](https://github.com/googleapis/java-spanner/pull/1870/files#r880579798) so when they initially weren't included in the Windows java 8 job, we started seeing the compilation error mentioned above (https://github.com/googleapis/java-spanner/issues/2205). IIRC, the `setJava` method only assigns the `PATH` to the `JAVA8_HOME` value but doesn't contain any logic to compute it. 

@harshachinta in the screenshot, I see that the windows tests are running in JDK 11. Are we no longer validating with JDK 8? If we're only testing with JDK 11 then we should no longer need these lines.

Reply:
@harshachinta Any thoughts on the last question?

Reply:
@mpeddada1 
I see that we are running tests on JDK 8 and JDK 11. https://github.com/googleapis/java-spanner/blob/fbf1df9f3fb12faaead8634b88fd4843cbdedf5b/.kokoro/build.sh#L26

Running on JDK 11 - https://screenshot.googleplex.com/87wspyzBrPNqRJE
Running on JDK 8 - https://screenshot.googleplex.com/uFos4MuHKEBYXNg

**Note:**
However, I observed that the JDK 8 and JDK 11 is already set but we are overwriting them by setting them again in the build.bat file.
https://github.com/googleapis/java-spanner/blob/fbf1df9f3fb12faaead8634b88fd4843cbdedf5b/.kokoro/build.bat#L20

From the below screenshot the environment variables `JAVA8_HOME` and `JAVA11_HOME` are already set.
https://screenshot.googleplex.com/7AUaPVjtWkS3ups
We are doing redundant work of downloading them again and in most cases leads to version mismatch due to the old version being hardcoded in the [build.bat](https://github.com/googleapis/java-spanner/blob/fbf1df9f3fb12faaead8634b88fd4843cbdedf5b/.kokoro/build.bat#L20) file

I think we no longer need these and can remove it. 
@olavloite @mpeddada1 Can you please let me know your views on this?

Reply:
Thanks for the detailed explanation @harshachinta! re `I see that we are running tests on JDK 8 and JDK 11.` and `the environment variables JAVA8_HOME and JAVA11_HOME are already set`: given this observation, it looks like we no longer need these environment variables.